AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreenViewModel.swift

```diff
@@ -231,8 +231,16 @@ class CompleteScreenViewModel: ObservableObject {
                     return
                 }
                 
+                // Special filter for Sensors as those measurements are not consistent
+                var streamWithMeasureementsCopy = downloadedStreamsWithMeasurements
+                guard let measurements = streamWithMeasureementsCopy.first?.measurements,
+                      let lastMeasurement = measurements.last else { return }
+                let twentyFourHours = 86400000 // 24 hours in miliseconds: 60 * 60 * 24
+                let beginingOfCorrectPeriod = lastMeasurement.time - twentyFourHours
+                streamWithMeasureementsCopy[0].measurements = (measurements.filter({ $0.time >= beginingOfCorrectPeriod }))
```

Comment:
Why are you doing this only for the first stream?

Reply:
My bad, I shouldn't